CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+### Documentation
+- Accessibility: improve introduction section @jurokapsiar ([#1368](https://github.com/stardust-ui/react/pull/1368))
```

Comment:
Please follow the order of changelog sectionsas described in INSTRUCTIONS above.